### DISCUSSION TITLE
Offline Install Python 3.6

### DISCUSSION THREAD

**Discussion by riaanvanniekerk** [duckdb/duckdb#8133] [Q&A]
I've been hitting a wall with shipping DuckDB's python package for offline installation on a RHEL system running python 3.6. Not a Pyrista so really hope someone here can point me in the right direction? If I can just grab everything I need to install offline that would be perfect, however the pip install continues to fail - even after everything in the development requirements were shipped with.

**Comment by ned2** [comment]
what kind of error messages etc do you get from the failed pip install?

**Reply by riaanvanniekerk** [reply]
G'day Ned,

It was initially complaining about multiple setuptools versions being used in conjunction (which I got around), then I had to deal w/ some gcc package maintenance, and only then did I get to the dependencies downloaded using pip and pip-compile (my poor man's solution) not being distribution compatible. This started w/ pybind11 but I turned around there. 

With fresh eyes I reckon the best course would be to hang tight for our next system upgrade and go w/ something like sqlite. I'll implement what's lacking myself as I don't use the entire duckdb feature set - mostly just the excellent & seamless file handling, which I doubt I'll hold a candle to but it's a bit of a crunch.

Thanks, 
Riaan
